HerPublicWebsite/wwwroot/css/_her-custom.scss

```diff
@@ -28,12 +28,12 @@
     display: flex;
     flex-direction: row;
     flex-wrap: wrap;
-    @include govuk-responsive-padding(4, "bottom");
 }
 
 .her_alphabet_links__letter {
-    width: 2em;
+    width: 8%;
```

Comment:
Gives a maximum of 12 letters per row (even without filtering not all letters are represented)